Sources/Widgets/Core/WidgetManager/vdom.js

```diff
@@ -35,6 +35,9 @@ export function render(vnode) {
     const key = keys[i];
     node.setAttribute(key, vnode.attrs[key]);
   }
+  Object.keys(vnode.eventListeners).forEach((key) => {
+    node.addEventListener(key, vnode.eventListeners[key]);
+  });
```

Comment:
At some point I want to move away from this vdom implementation to something like [snabbdom](https://github.com/snabbdom/snabbdom). That way we get proper handling of event listeners, including proper removal of event listeners. Also so we don't have to maintain our own vdom.

For now, this is fine, but the current vdom doesn't address event listener removal when the incoming virtual node doesn't have the same set of listeners.

Reply:
gotcha. I'll add a note in the code and merge it for the time being.